actions/prepare_branch.go

```diff
@@ -50,18 +50,9 @@ func (ctx PrepareBranch) MkReleaseLog() error {
 	filename := fmt.Sprintf("docs/changelogs/%s.md", ctx.Version.MajorMinor())
 	branch := repos.Kubo.VersionReleaseBranch(ctx.Version)
 
-	name := os.Getenv("GITHUB_USER_NAME")
-	if name == "" {
-		name = "Kubo Releaser"
-	}
-	email := os.Getenv("GITHUB_USER_EMAIL")
-	if email == "" {
-		email = "noreply+kuboreleaser@ipfs.tech"
-	}
-	token := os.Getenv("GITHUB_TOKEN")
-	if token == "" {
-		return fmt.Errorf("GITHUB_TOKEN not set")
-	}
+	name := util.GetenvPrompt("GITHUB_USER_NAME")
+	email := util.GetenvPrompt("GITHUB_USER_EMAIL")
+	token := util.GetenvPromptSecret("GITHUB_TOKEN", "The token should have the following scopes: ... Please enter the token:")
```

Comment:
`...` is a TODO?

Reply:
Yeap, I didn't check what those should be exactly yet.